File: JavaScript_Phase_Gate/Day2/Task4.js
function isPossible(gallonLeft){
	const distance = 50;
	let milesPerGallon = 25;
	if (milesPerGallon * gallonLeft == distance)
		return true;
	else
		return false;
}


let gallonAvailable = 2
console.log(isPossible(gallonAvailable));

File: JavaPhaseGate/Day2/Task4.java
public class Task4{
public static void main(String[] args){


int gallonAvailable = 2;
System.out.print(isPossible(gallonAvailable));


}
public static boolean isPossible(int gallonLeft){
	int distance = 50;
	int milesPerGallon = 25;
	if (milesPerGallon * gallonLeft == distance)
		return true;
	else
		return false;
}

}
